apps/nightscout/nightscout.star

```diff
@@ -13,143 +13,598 @@ load("encoding/json.star", "json")
 load("cache.star", "cache")
 load("schema.star", "schema")
 load("re.star", "re")
+load("humanize.star", "humanize")
+load("sunrise.star", "sunrise")
 
-COLOR_RED = "#f00"
-COLOR_YELLOW = "#ff0"
-COLOR_GREEN = "#0f0"
-COLOR_GREY = "#666"
+COLOR_RED = "#C00"
+COLOR_DARK_RED = "#911"
+COLOR_YELLOW = "#ff8"
+COLOR_ORANGE = "#d61"
+COLOR_GREEN = "#2b3"
+COLOR_GREY = "#777"
 COLOR_WHITE = "#fff"
+COLOR_NIGHT = "#444"
 
-DEFAULT_NORMAL_HIGH = 150
+DEFAULT_NORMAL_HIGH = 180
 DEFAULT_NORMAL_LOW = 100
 DEFAULT_URGENT_HIGH = 200
 DEFAULT_URGENT_LOW = 70
 
-CACHE_TTL_SECONDS = 300
+DEFAULT_SHOW_GRAPH = "true"
+DEFAULT_SHOW_CLOCK = "true"
+DEFAULT_NIGHT_MODE = "false"
+GRAPH_WIDTH = 43
+GRAPH_BOTTOM = 50
+GRAPH_TOP = 275
+
+CACHE_TTL_SECONDS = 0
```

Comment:
This will actually be interpreted as "no ttl provided", causing the default TTL to be used instead. Better approach is to completely remove the cache logic.

Reply:
Updated this functionality to use a 30 min cache but only in cases of Nightscout error. 

Use live data when able.
If error, use cache.
If error and no cache, display error.